index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Webpage</title>
+    
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <a class="nav-link" href="#profile-link"> profile link </a>
```

Comment:
* Remember to explain why you request a change - it might be obvious for you but not for someone else. 
* Give the reason why using `class` is better than using `id` this would give a better understanding to why the change was required